contrib/babelfishpg_tsql/src/pl_handler.c

```diff
@@ -4339,34 +4358,28 @@ pltsql_call_handler(PG_FUNCTION_ARGS)
 		PG_CATCH();
 		{
 			set_procid(prev_procid);
-			/* Decrement use-count, restore cur_estate, and propagate error */
 			pltsql_trigger_depth = save_pltsql_trigger_depth;
-			func->use_count--;
-			func->cur_estate = save_cur_estate;
-			pltsql_remove_current_query_env();
-			pltsql_revert_guc(save_nestlevel);
-			pltsql_revert_last_scope_identity(scope_level);
-			terminate_batch(true /* send_error */ , false /* compile_error */ );
```

Comment:
here we had set send_error = true
which re throws the error in terminate batch, doesn't that mean we never actually hit the next two lines in catch block ?
code coverage seems to indicate so.
cc: @surendravishnoi